sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -3210,4 +3211,183 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
       messageParameters = Map("expression" -> toSQLExpr(expression))
     )
   }
+
+  def cannotConvertProtobufTypeToSqlTypeError(
+    protobufColumn: String,
+    sqlColumn: String,
+    protobufType: String,
+    sqlType: String): Throwable = {
+    new AnalysisException(
+      errorClass = "_LEGACY_ERROR_TEMP_2251",
```

Comment:
We use the prefix `_LEGACY_ERROR_TEMP_` to convert existing exception to error classes but for new exception, please, assign appropriate names to error classes.